lib/generated/intl/messages_en.dart

```diff
@@ -74,6 +74,12 @@ class MessageLookup extends MessageLookupByLibrary {
         "about": MessageLookupByLibrary.simpleMessage("About"),
         "account_suspended":
             MessageLookupByLibrary.simpleMessage("Account suspended"),
+        "activate_non_confirmation_bias_mode_description":
+            MessageLookupByLibrary.simpleMessage(
+                "Don\'t show the author of the tweet on a post. Avoid confirmation bias based on authoritative arguments."),
```

Comment:
I think this description is a bit long, and probably won't fit on one line on most devices. Maybe truncate it to `Don't show tweet authors`, as the title declares it as "non-confirmation bias mode" already

Reply:
This one's still outstanding, I think

Reply:
Take a look at the `intl_en.arb` file, the `messages_en.dart` is generated based on it.